routes/users_routes.js

```diff
@@ -235,7 +236,7 @@ router.post(
           } else {
             User.findOne(
               {
-                email: req.body.email,
+                email: recoveryEmail,
```

Comment:
This is the only one I wasn't 100% with - should we use the email exactly how the user types it or downcast it? Not sure if `findOne` is case-insensitive under the covers.

Reply:
We should use findOne without the regex, as it is O(1) vs O(n). As long as we consistently normalize on write and on read, it won't be an issue

Reply:
Makes sense! 

Not sure how long the writes have been normalized to lowercase (new contributor), but for any user that somehow registered without being normalized to lower case would be unable to recover their password following this?

Reply:
No, the login recovery documents are only valid for 15 min. We already normalize emails for the User table